addons/common/functions/fnc_exportMissionSQF.sqf

```diff
@@ -110,11 +110,17 @@ private _fnc_processGroup = {
 private _fnc_processInventory = {
     params ["_object", "_varName"];
 
-    if (_object call FUNC(hasDefaultInventory)) exitWith {};
-
     if (_object isKindOf "CAManBase") then {
-        _outputObjects pushBack ["%1 setUnitLoadout %2;", _varName, getUnitLoadout _object];
+        _outputObjects pushBack "[{";
+        _outputObjects pushBack "    params [""_unit""];";
+        if !(_object call FUNC(hasDefaultInventory)) then {
+            _outputObjects pushBack ["    _unit setUnitLoadout %1;", getUnitLoadout _object];
+        };
+        _outputObjects pushBack "    _unit call BIN_fnc_CBRNHoseInit;";
+        _outputObjects pushBack ["}, [%1]] call CBA_fnc_execNextFrame;", _varName];
```

Comment:
I think we should decide whether we want export mission SQF to require any mods (CBA in this case) to work. If so, we could then change portions of the code to just call ZEN function as well.

Reply:
I forgot about that. I can easily swap it with `BIS_fnc_addStackedEventHandler`.